contracts/protektCore/pToken.sol

```diff
@@ -7,6 +7,8 @@ import "@openzeppelin/contracts/token/ERC20/SafeERC20.sol";
 import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
 import "@openzeppelin/contracts/token/ERC20/ERC20Detailed.sol";
 import "./helpers/HarvestRewardsCompoundDaiManual.sol";
+import "../claimsManagers/interfaces/IClaimsManagerCore.sol";
+import "../claimsManagers/ClaimsManagerSingleAccount.sol";
```

Comment:
@stephenbyrne99 I don't think you need to import this contract. I believe the interface should be enough. See what you think.